lib/soak/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # BUILDER
 #
 FROM docker.io/rust:1.58-buster as builder
-WORKDIR vector
+WORKDIR /vector
```

Comment:
I assume this means that the workdir is by default `/`?

Reply:
Correct - https://docs.docker.com/engine/reference/run/#workdir